.ibm/images/Dockerfile

```diff
@@ -1,103 +1,62 @@
+    OCM_VERSION="0.1.76" \
+    GO_VERSION=1.19 \
+    GOPATH=/go \
+    PATH=$GOPATH/bin:/usr/local/go/bin:$PATH
+
+# Install essential dependencies and Node.js tools
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    nodejs-dev nodejs \
+    openssl libssl-dev ca-certificates \
+    make cmake cpp gcc g++ zlib1g zlib1g-dev brotli libbrotli-dev \
+    python3 sshpass jq colorized-logs rsync curl wget && \
+    npm install -g typescript yarn@latest && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+    echo "Node: $(node -v)\nNPM: $(npm -v)\nYarn: $(yarn -v)\nTypeScript: $(tsc -v)"
+
+# Ensure the current user is root and verify permissions
 RUN echo "whoami: $(whoami)" \
-    # command "id" should print:
-    # uid=0(root) gid=0(root) groups=0(root)
-    # which means the current user is root
+    # The "id" command should confirm the user is root
     && id \
-    && npm install -g typescript \
-    # give every user read access to the "/root" folder where the binary is cached
-    # we really only need to worry about the top folder, fortunately
-    && ls -la /root \
+    # Make the "/root" folder readable for all users
     && chmod 755 /root \
-    # always grab the latest Yarn
-    # otherwise the base image might have old versions
-    # NPM does not need to be installed as it is already included with Node.
-    && npm i -g yarn@latest \
-    # Show where Node loads required modules from
+    # Check where Node.js loads required modules
     && node -p 'module.paths'
-# plus Electron and bundled Node versions
-
-RUN  echo  " node version:    $(node -v) \n" \
-    "npm version:     $(npm -v) \n" \
-    "yarn version:    $(yarn -v) \n" \
-    "typescript version:  $(tsc -v) \n" \
-    "debian version:  $(cat /etc/debian_version) \n" \
-    "user:            $(whoami) \n"
 
-RUN curl -fsSL https://clis.cloud.ibm.com/install/linux | sh && \
-    curl -sLO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
-    chmod +x kubectl && \
-    mv kubectl /usr/local/bin/ && \
-    apt-get update -y && \
-    apt-get install -y sshpass jq colorized-logs && \
-    rm -rf /var/lib/apt/lists/*
-
-# Set Go version and the expected SHA256 hash for verification
-ENV GO_VERSION 1.19
-ENV GO_SHA256 464b6b66591f6cf055bc5df90a9750bf5fbc9d038722bb84a9d56a2bea974be6
-
-# Install Go and other tools used by the pipeline
-RUN apt-get update && \
-    apt-get install -y curl && \
-    curl -LO "https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz" && \
-    echo "${GO_SHA256} go${GO_VERSION}.linux-amd64.tar.gz" | sha256sum -c - && \
+# Install Go, Helm, OpenShift CLI, ocm-cli, and yq
+RUN curl -LO "https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz" && \
+    echo "464b6b66591f6cf055bc5df90a9750bf5fbc9d038722bb84a9d56a2bea974be6 go${GO_VERSION}.linux-amd64.tar.gz" | sha256sum -c - && \
```

Comment:
I think that the `GO_SHA256` should be kept as a variable close to `GO_VERSION`. You would be updating them both together. The comment `# Set Go version and the expected SHA256 hash for verification` seems to be useful and I would keep it around.